### DISCUSSION TITLE
Why startship worked fine only the first time after installed?

### DISCUSSION THREAD

**Discussion by callmehanu** [starship/starship#4767] [Q&A] [answered]
I'm using Fedora 37 WS and Alacritty (0.12.0-dev) as terminal emulator.
I installed Starship as doc recommend, but only worked great after installed it.
I closed Alacritty, run it again, and I lost Starship prompt.

To install Starship I follow this steps:
1. `curl -sS https://starship.rs/install.sh | sh`
2. `eval "$(starship init bash)"`

Just that! 

I'm new on linux and also on development, so any advice would be welcome.

**Comment by matchai** [comment, marked as answer]
Please be sure to add the second line to your shell's initialization file (e.g. `~/.zshrc` for zsh)
Simply running it instead of adding it to the init file will do as you described, only initialize Starship that shell instance.

**Reply by callmehanu** [reply]
@matchai great... really thanks for your respond.
I am using bash... when do you mean _shell's initialization file_ do you mean `.bashrc` in my case?

**Comment by callmehanu** [comment]
Done! .... Finally Starship works fine! 
Thanks!

**Comment by callmehanu** [comment]
Anyway, do you consider this detail (edit the shell initialization file) should appear in the doc? @matchai

**Reply by matchai** [reply]
It seems to be documented already:
<img width="884" alt="CleanShot 2022-12-28 at 13 49 33@2x" src="https://user-images.githubusercontent.com/4658208/209759238-c4ddb779-6461-4a8b-bf3a-1c6eea6d796a.png">
